### PR TITLE
Rework combined project stats fetching

### PR DESCRIPTION
**Pull request by IvanIvanoff**
Make smaller batches with higher concurrency so they can be calculated
in parallel

#### Summary

[//]: # (#### Related PRs)

[//]: # (#### Additional deploy notes)

[//]: # (#### Screenshots)

### CODE REVIEW

**Comment by tspenov** [comment]
@IvanIvanoff I don't know why it is not showing any conflicts with master but there were some changes in concurrency and chunks for TOTAL_ERC20 that are rolled back here.

**Comment by IvanIvanoff** [comment]
@tspenov The ERC20 code only calls one of the changed functions here - nothing else

**Comment by tspenov** [comment]
@IvanIvanoff 
There is no point in chunking and spawning and then again chunking and spawning. The function can be reduced to:
```elixir
  def fetch_prices_with_resolution("TOTAL_ERC20", from, to, resolution) do
    measurements =
      Sanbase.Model.Project.List.erc20_projects()
      |> Enum.map(&Sanbase.Influxdb.Measurement.name_from/1)


    fetch_combined_mcap_volume(
      measurements,
      from,
      to,
      resolution
    )
  end
```

File: lib/sanbase/prices/store.ex [hunk 1]
```diff
@@ -326,48 +342,64 @@ defmodule Sanbase.Prices.Store do
...
-    result = Enum.zip([slugs, volume_values, marketcap_values, marketcap_percent])
+        {:ok, data}
 
-    {:ok, result}
+      %{errors: [error | _]} ->
+        {:error, error}
+    end
   end
 
   defp volume_mcap_multiple_measurements_reducer(_, _), do: {:error, nil}
```
Comment (tspenov) [inline lib/sanbase/prices/store.ex]: This clause is never reached.

**Comment by tspenov** [comment]
@IvanIvanoff please, resolve conflicts

**Pull request merged**